src/general/MainMenu.cs

```diff
@@ -273,6 +283,24 @@ public override void _Process(float delta)
                     }
                 }
             }
+
+            // Low menu performance will never be warned about if the popup has been dismissed,
+            // if 3D backgrounds have been disabled, if the popup has been shown but not dismissed
+            // on this menu session, or if the max framerate is set to 30
+            // In addition, tracking only begins after one second in the menu
+            if (!Settings.Instance.IsNoticePermanentlyDismissed(DismissibleNotice.LowPerformanceWarning)
+                && Settings.Instance.Menu3DBackgroundEnabled && canShowLowPerformanceWarning
+                && Settings.Instance.MaxFramesPerSecond != 30)
```

Comment:
```suggestion
                && Settings.Instance.MaxFramesPerSecond > 30)
```

I think this would futureproof the code if we add an extremely low framerate option in the future.